src/Ocelot/Cache/CachedResponse.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using System.Net;
+
+namespace Ocelot.Cache
+{
+	public class CachedResponse
+	{
+		public CachedResponse()
+		{
+			StatusCode = HttpStatusCode.OK;
+			Headers = new Dictionary<string, IEnumerable<string>>();
+			Body = "";
+		}
+
+		public HttpStatusCode StatusCode { get; set; }
```

Comment:
Can we have private setters here please, the project only has public setters for the file configuration stuff because microsofts framework for config cant support private setters.

Reply:
yep, sure no problem!